app/models/sample.rb

```diff
@@ -93,6 +93,11 @@ class Sample < ApplicationRecord
   # setter
   attr_writer :bulk_mode
 
+  enum pipeline_type: {
```

Comment:
We should not duplicate this definition.... should use the same map as pipeline run.